emu/src/cpu/single_data_transfer.rs

```diff
@@ -32,7 +32,7 @@ impl From<ArmModeOpcode> for SingleDataTransfer {
 
 /// There two different kind of write or read for memory.
 #[derive(Default)]
-enum ReadWriteKind {
+pub enum ReadWriteKind {
```

Comment:
Yes we can create a module of `OperationsFlag`? Like pre/post load/store and so on...

Reply:
Done!